tutor-multi-chart/Chart.yaml

```diff
@@ -16,7 +16,13 @@ appVersion: "14.1.1"
 dependencies:
 # This is just info for the "helm dependency update" command, which will update the ./charts/ directory when run, using
 # this information.
-# We use Traefik as an HTTP(S) server, reverse proxy, and load balancer for all of 
+# We use Traefik as an HTTP(S) server, reverse proxy, and load balancer for all of
 - name: traefik
   version: "20.3.0"
   repository: https://traefik.github.io/charts
+
+- name: elasticsearch
```

Comment:
As discussed in the call yesterday - we may need to change this to OpenSearch or something else in the future.

Reply:
This configuration for `OpenSearch` is similar, but not the same. At minimum the names of the config values are OpenSearch and not ElasticSearch. Configuring users might also be different, but I didn't check in detail.

I'll create a dedicated ticket for it.